functional-tests/network-deployment/miner_workflow_test.go

```diff
@@ -1,186 +0,0 @@
-package networkdeployment_test
-
-import (
-	"bytes"
-	"context"
-	"crypto/rand"
-	"io"
-	"io/ioutil"
-	"math/big"
-	"testing"
-	"time"
-
-	"github.com/ipfs/go-ipfs-files"
-
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-
-	"github.com/filecoin-project/go-filecoin/internal/pkg/protocol/storage/storagedeal"
-	tf "github.com/filecoin-project/go-filecoin/internal/pkg/testhelpers/testflags"
-	"github.com/filecoin-project/go-filecoin/internal/pkg/types"
-	minerActor "github.com/filecoin-project/go-filecoin/internal/pkg/vm/actor/builtin/miner"
-	"github.com/filecoin-project/go-filecoin/internal/pkg/vm/address"
-	"github.com/filecoin-project/go-filecoin/tools/fast"
-	"github.com/filecoin-project/go-filecoin/tools/fast/fastesting"
-	"github.com/filecoin-project/go-filecoin/tools/fast/series"
-)
-
-func TestMinerWorkflow(t *testing.T) {
-	network := tf.DeploymentTest(t)
```

Comment:
This test was designed to run against live networks to verify automated deploy processes. Just noting that we'll probably want to bring something back later (but it'll probably look pretty different). cc @ZenGround0 @travisperson